dsl/src/main/scala/org/http4s/dsl/RequestDsl.scala

```diff
@@ -40,6 +40,7 @@ trait RequestDsl extends Methods with Auth {
   val IntVar: impl.IntVar.type
   val LongVar: impl.LongVar.type
   val UUIDVar: impl.UUIDVar.type
+  val StringVar: impl.StringVar.type
```

Comment:
This is not backwards-compatible: we cannot add an abstract method to an open `trait`, since what happens to all the existing implementations that are missing this one?

Reply:
Uh that's annoying. I'm don't know what could be done in this direction that doesn't end up changing RequestDsl.

Reply:
So I don't know anything about this corner of http4s and why it's written this way, but this would fix the compat issues:
```suggestion
  def StringVar: impl.StringVar.type = impl.StringVar
```

Reply:
I was lazy and didn't check it, I'll again later and avoid the CI spam